src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -65,9 +65,9 @@ public function __construct(IriConverterInterface $iriConverter, DataPersisterIn
         $this->validator = $validator;
     }
 
-    public function __invoke(string $resourceClass = null, string $rootClass = null, string $operationName = null): callable
```

Comment:
I would revert this change. This is triggered for queries to, and queries aren't mutations. Mutations + Queries = operations.